### ISSUE TITLE
Receive "Command failed: Read timeout on endpoint URL"

### ISSUE THREAD

**Issue by anton-kazanov** [datahub-project/datahub#8986]
**Describe the bug**
When I try to ingest some data from S3 bucket I receive the timeout error "Command failed: Read timeout on endpoint URL". 

**To Reproduce**
1. Go to the Datahub UI
2. Click on the "Ingestion" in the upper right
3. Click "Create new source"
4. In the source list click "Other"
5. Use this template to tune your ingestion from the source:

transformers:
    -
        type: simple_add_dataset_ownership
        config:
            owner_urns:
                - 'urn:li:corpuser:<username>'
            ownership_type: PRODUCER
    -
        type: simple_add_dataset_tags
        config:
            tag_urns:
                - 'urn:li:tag:<needed_tag>'
sink:
    type: datahub-rest
    config:
        server: '<URL_of_your_Datahub_GMS>'
        token: '${name_of_the_token}'
source:
    type: s3
    config:
        platform_instance: <name_of_the_platform_instance>
        profiling:
            enabled: false
        path_specs:
            -
                include: 's3://<path_to_S3_bucket>/{table}/{partition[0]}/{partition[1]}/{partition[2]}/*.*'
        env: <>
        aws_config:
            aws_endpoint_url: 'endpoint_URL'
            aws_access_key_id: <access_key>
            aws_region: <region>
            aws_secret_access_key: '${secret_access_key}'

**Expected behavior**
All of the data are ingested from the source properly.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Debian GNU/Linux 12 (inside the Pod with running Datahub)
 - Version Datahub v0.11.0 (inside the Pod with running Datahub)

**Additional context**

**Comment by anton-kazanov** [comment]
Greetings.

It seems that the bug is no more.
What I do to solve it:
1) Need to understand in which directory the Datahub installs the necessary software (you can see this in new ingestion log).
2) Find in this directory the file "./lib/python3.<your_python_version>/site-packages/datahub/ingestion/source/aws/aws_common.py"
3) Run the following sed command on this file:
`sed -i 's/config=Config(proxies=self.aws_proxy)/config=Config(proxies=self.aws_proxy, read_timeout=1800)/g' ./lib/python3.<your_python_version>/site-packages/datahub/ingestion/source/aws/aws_common.py`
4) Run the ingestion through the Datahub UI to check if it works.

To sum up you only need to add the `read_timeout=1800` option into the aws_common.py lib inside the datahub venv dir.
Note: I think it's not necessary to use exactly 1800 value. Maybe in your configuration will work another value (100, 200 or 300).

**Comment by anton-kazanov** [comment]
Maybe in the next releases of Datahub there will be an option for ingestion code?
So, everyone can choose the appropriate timeout for their ingestion.

**Comment by YuriyGavrilov** [comment]
https://github.com/datahub-project/datahub/issues/8733 will be closed due to founded solution
